webpack/ForemanTasks/Components/TasksTable/TasksTableConstants.js

```diff
@@ -0,0 +1,8 @@
+import { getControllerSearchProps } from 'foremanReact/constants';
+
+export const TASKS_CONTROLLER = 'tasks';
+export const TASKS_SUCCESS = 'TASKS_SUCCESS';
```

Comment:
The convention is to have long const names, e.g. `TASKS_TABLE_CONTROLLER`, `TASKS_TABLE_GET_ITEMS_SUCCESS`.

Reply:
`TASKS_TABLE_GET_ITEMS_SUCCESS` can be generated using:
https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/common/table/actionsHelpers/actionTypeCreator.js#L1

Reply:
Changed to TASKS_TABLE_ID = TASKS_TABLE for clarity